README.md

```diff
@@ -115,14 +136,14 @@ The `jlpm` command is JupyterLab's pinned version of
 `yarn` or `npm` in lieu of `jlpm` below.
```

Comment:
I don't think this is true anymore. I had to use the bundled yarn version that `jlpm install` uses.

Reply:
Hmm, let's check the upstream extensions cookie-cutter (but in general I recall the version always being fairly important).